app/models/agents/lifx_pulse_lights_agent.rb

```diff
@@ -0,0 +1,64 @@
+module Agents
+  class LifxPulseLightsAgent < Agent
```

Comment:
Each API action is different enough in the arguments it takes that I was planning to have an agent for each of the interesting endpoints to keep the agent form approachable. That means there will be about 4 LIFX agents. Is there a better plan? I'd like to have a more intelligent form that presents different subsets of options based on the action the user selects from a dropdown, but I don't see a way for FormConfigurable to support that.